ClientTests/MockProfile.swift

```diff
@@ -111,7 +111,7 @@ public class MockProfile: Profile {
         return CertStore()
     }()
 
-    lazy var bookmarks: protocol<BookmarksModelFactorySource, SyncableBookmarks, LocalItemSource, MirrorItemSource, ShareToDestination> = {
+    lazy var bookmarks: protocol<BookmarksModelFactorySource, SyncableBookmarks, LocalItemSource, BufferedBookmarks, MirrorItemSource, ShareToDestination> = {
```

Comment:
Oops! I've been running the tests from a new SyncTests target and forgot to run the client target :(





On Sat, May 28, 2016 at 8:58 AM -0700, "Richard Newman" <notifications@github.com> wrote:












In ClientTests/MockProfile.swift:
> @@ -111,7 +111,7 @@ public class MockProfile: Profile {
>          return CertStore()
>      }()
>  
> -    lazy var bookmarks: protocol<BookmarksModelFactorySource, SyncableBookmarks, LocalItemSource, MirrorItemSource, ShareToDestination> = {
> +    lazy var bookmarks: protocol<BookmarksModelFactorySource, SyncableBookmarks, LocalItemSource, BufferedBookmarks, MirrorItemSource, ShareToDestination> = {



Don't forget to run tests, then fix this when they fail :)



—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.